ledger/sandbox-common/BUILD.bazel

```diff
@@ -27,6 +27,7 @@ da_scala_library(
     deps = [
         "//daml-lf/data",
         "//language-support/scala/bindings",
+        "//ledger-service/cli-opts",
```

Comment:
Probably `cli-opts` belongs in a more "neutral" directory. Having the sandbox depend on something under the JSON API sub-tree is valid but feels a bit weird.

Reply:
Happy to move this somewhere else. How about `//libs-scala/cli-opts`? That would also be an opportunity to split up http, tls, and jwt into dedicated Bazel targets. Though, in that case I'd prefer to do this in a follow-up PR.

Reply:
I've always seen `//libs-scala` as more of a repository for libraries that are not necessarily related to the ledger. Maybe under `//ledger`?

Reply:
@gerolf-da @cocreature Feel free to shut me down on this, maybe I'm being too nit-picking here. 😉

Reply:
I'll merge this one as is as the increased test timeout will help with CI stability on master and will look into a better label in a follow up

Reply:
🙇🏻